### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by hoaiphu99**
Change from Antd Design to NextUI
Home page NextUI: https://nextui.org/

### CODE REVIEW

File: src/api/utubeDownloaderApi.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios'
+
+export async function getTitleVideo(url: string) {
+  const data = await axios
+    .post(`https://ltools-be.onrender.com/api/utube/getTitle`, { url })
```
Comment (vht2023) [inline src/api/utubeDownloaderApi.ts:5]: should create constant variable for url and endpoint of API

File: src/api/utubeDownloaderApi.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    data
+  )
+
+  return data.title
+}
+
+export async function downloadVideo(url: string) {
+  const { data } = await axios
+    .post(
+      `https://ltools-be.onrender.com/api/utube/download`,
```
Comment (vht2023) [inline src/api/utubeDownloaderApi.ts:19]: should create constant variable for url and endpoint of API

File: src/app/about/page.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 const About = () => {
-  return <div>About</div>
+  return <div>Team chúng tôi có 2 thành viên: KT, Toryo</div>
```
Comment (vht2023) [inline src/app/about/page.tsx:4]: t tên Thiên, nickname: thiendumprovipabcxyz

**Review by vht2023 — APPROVED** [review]

File: src/api/utubeDownloaderApi.ts (hunk 1 above)
Comment (hoaiphu99) [reply src/api/utubeDownloaderApi.ts:5]: Ok bro, will define later

File: src/app/about/page.tsx (hunk 3 above)
Comment (hoaiphu99) [reply src/app/about/page.tsx:4]: OK thiendumprovipabcxyz =)))

**Pull request merged**